solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -32,6 +32,18 @@ interface ISortitionPool {
     function isOperatorInPool(address operator) external view returns (bool);
 
     function isOperatorEligible(address operator) external view returns (bool);
+
+    // TODO: Temporary function for testing only. Not sure how the removal
+    //       mechanism will work at the moment.
+    function removeOperators(address[] memory operators) external;
```

Comment:
Yeah, that's clear now. This TODO comes from times where I wasn't sure about that. The signature of this method has been changed as part of https://github.com/keep-network/keep-core/pull/2664/commits/c7cf1edf952ebb948e413e5709b6c050db041600

Regarding tests against a real pool, I agree. However, I'd love to introduce them after this PR and #2671 land on `main`. This will make things far easier given the fact the whole beacon codebase is under heavy development and things change very quickly which creates conflicts. Also, I think making gas optimizations on `main` code would be easier than making them on stacked PRs.

Regarding TODO about `keep-network/sortition-pools`, added it within: https://github.com/keep-network/keep-core/pull/2664/commits/b89888d0e34c8a85acc5adb7180c3bbfdceb082e